src/Jobs/ResendMailJob.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Vormkracht10\Mails\Jobs;
+
+use Illuminate\Broadcasting\InteractsWithSockets;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Mail\Message;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Vormkracht10\Mails\Models\Mail;
+
+class ResendMailJob implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, InteractsWithSockets, Queueable, SerializesModels;
+
+    public function __construct(
+        private readonly Mail $mail,
+        private array $to = [],
+        private array $cc = [],
+        private array $bcc = [],
+    ) {
+        $this->checkFields($this->mail);
+    }
+
+    public function handle(): void
+    {
+        \Illuminate\Support\Facades\Mail::send([], callback: fn (Message $mail) => $mail
```

Comment:
Deze class importen i.p.v. met \, je kunt dit doen door de 2e "Mail" class te hernoemen met "as" in de import.